examples/01b-Getting-started-Pytorch.ipynb

```diff
@@ -335,7 +335,9 @@
     "optim = Adam\n",
```

Comment:
<p>Does the dataloader keep the dataset on CPU or GPU? How can we initialize the dataloader on CPU? Can we add a description and/or link to the documentation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I've added a comment about this above this line. Can update for clarify. The default device to load into is determined based on availabliliy of GPU + if you have cudf installed. </p><p> </p><p>Which is  determined by the value of <code>merlin.core.dispatch.HAS_GPU</code>   The logic is here in the dataloader:</p><p> </p><p>https://github.com/NVIDIA-Merlin/dataloader/blob/v0.0.3/merlin/loader/loader_base.py#L78-L81 </p><p> </p><pre class="ql-syntax" spellcheck="false"># the dataloader determines whether or not to load data onto GPU or CPU
# depending on whether you have a GPU available and cudf installed
</pre>